dspace-spring-rest/src/test/java/org/dspace/app/rest/InitializeEntitiesIT.java

```diff
@@ -0,0 +1,155 @@
+
+        getClient().perform(get("/api/core/relationshiptypes"))
+
+                   //We expect a 200 OK status
+                   .andExpect(status().isOk())
+                   //The type has to be 'discover'
+                   .andExpect(jsonPath("$.page.totalElements", is(10)))
+                   //There needs to be a self link to this endpoint
+                   .andExpect(jsonPath("$._links.self.href", containsString("api/core/relationshiptypes")))
+                   //We have 4 facets in the default configuration, they need to all be present in the embedded section
+                   .andExpect(jsonPath("$._embedded.relationshiptypes", containsInAnyOrder(
+                       RelationshipTypeMatcher.matchRelationshipTypeEntry(relationshipTypes.get(0)),
+                       RelationshipTypeMatcher.matchRelationshipTypeEntry(relationshipTypes.get(1)),
+                       RelationshipTypeMatcher.matchRelationshipTypeEntry(relationshipTypes.get(2)),
+                       RelationshipTypeMatcher.matchRelationshipTypeEntry(relationshipTypes.get(3)),
+                       RelationshipTypeMatcher.matchRelationshipTypeEntry(relationshipTypes.get(4)),
+                       RelationshipTypeMatcher.matchRelationshipTypeEntry(relationshipTypes.get(5)),
+                       RelationshipTypeMatcher.matchRelationshipTypeEntry(relationshipTypes.get(6)),
+                       RelationshipTypeMatcher.matchRelationshipTypeEntry(relationshipTypes.get(7)),
+                       RelationshipTypeMatcher.matchRelationshipTypeEntry(relationshipTypes.get(8)),
+                       RelationshipTypeMatcher.matchRelationshipTypeEntry(relationshipTypes.get(9)))
+                   ));
+
+        getClient().perform(get("/api/core/relationshiptypes/" + relationshipTypes.get(0).getID()))
+                   .andExpect(status().isOk())
+                   .andExpect(jsonPath("$.leftMinCardinality", is(0)));
+        String pathToFile = configurationService.getProperty("dspace.dir") +
+            File.separator + "config" + File.separator + "entities" + File.separator + "relationship-types-update.xml";
+        runDSpaceScript("initialize-entities", "-f", pathToFile);
+
+        RelationshipType alteredRelationshipType = relationshipTypes.get(0);
+        alteredRelationshipType.setLeftMinCardinality(10);
+        getClient().perform(get("/api/core/relationshiptypes"))
+
+                   //We expect a 200 OK status
+                   .andExpect(status().isOk())
+                   //The type has to be 'discover'
+                   .andExpect(jsonPath("$.page.totalElements", is(10)))
+                   //There needs to be a self link to this endpoint
+                   .andExpect(jsonPath("$._links.self.href", containsString("api/core/relationshiptypes")))
+                   //We have 4 facets in the default configuration, they need to all be present in the embedded section
```

Comment:
This comment is also incorrect.  Could we better describe this test in general. It's not straightforward what we are looking for... are we simply verifying running this command twice results in a no-op?  Are we looking for specific changes?  This test needs some better comments in general.